cuckoo/table_test.go

```diff
@@ -2,53 +2,63 @@ package cuckoo
 
 import (
 	"bytes"
+	"encoding/binary"
 	"fmt"
 	"math/rand"
 	"strconv"
 	"testing"
 )
 
+const ItemSize = uint64(64)
```

Comment:
this will be exported from the package even though in test file. should be lower case